README.md

```diff
@@ -1 +1,62 @@
-# Math-Magicians
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math-Magicians
+
+> Math Magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations and read a random math-related quote.
+
+
+## Built With
+
+- HTML5
+- CSS3
+- JavaScript (ES6)
+- React
+- Linters (Stylelint, Eslint)
+- Gitflow
+- Jest
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- A web browser 
+- A code editor
+
+### Setup
+
+Assuming you have git and node.js installed on your computer, please follow the steps below: 
+
+- Clone the repository using the following command: `git clone https://github.com/laiifuu/math-magicians.git`
+- Navigate to that folder in the command line using the `cd` command.
+- Run the following command to install the necessary packages: `npm install`. 
+- Run the following command to view the project in the browser: `npm start`. 
+
+## Authors
+
+👤 **Hayet Fellah**
+
+- GitHub: [@laiifuu](https://github.com/laiifuu)
+- Twitter: [@hayet_fe](https://twitter.com/hayet_fe)
+- LinkedIn: [Hayet Fellah](https://www.linkedin.com/in/hayet-f-5b4347247)
+
+## 🤝 Contributing
+
```

Comment:
- [ ] [optional] It would be nice to add a link to your issues page, for easier contribution by visitors to your repo.